public/app/percona/settings/components/Platform/Connected/ModalBody/ModalBody.messages.ts

```diff
@@ -0,0 +1,9 @@
+export const Messages = {
+  modalBodyPlatformUser:
+    'Are you sure you want to disconnect from Percona Platform? This will automatically log you out from PMM.',
+  modalBody:
+    'Are you sure you want to force-disconnect this PMM instance? This will unlink the instance from its current organization and stop all synchronizations with Percona Platform.',
```

Comment:
```suggestion
    'Are you sure you want to force disconnect this PMM instance? This will unlink the instance from its current organization and stop all synchronizations with Percona Platform.',
```